use_cases/gretel.json

```diff
@@ -329,7 +329,7 @@
       "icon": "transform.png",
       "modelType": "transform",
       "modelCategory": "transform",
-      "defaultConfig": "config_templates/gretel/transform/default.yml",
+      "defaultConfig": "config_templates/gretel/transform/transform-v1-default.yml",
```

Comment:
This change is not needed. We want to use Tv2 for NER. It's the same config as for structured data. We're just using different sample datasets.

Reply:
ok. the modelType on this one is wrong then.

Reply:
updated the model type